src/main/java/xbot/common/subsystems/vision/AprilTagVisionSubsystem.java

```diff
@@ -39,8 +42,9 @@
  * Based on the AdvantageKit sample implementation by team 6328.
  */
 @Singleton
-public class AprilTagVisionSubsystem extends SubsystemBase {
+public class AprilTagVisionSubsystem extends SubsystemBase implements DataFrameRefreshable {
```

Comment:
⭐ since we might be doing a lot of iterating on this code in the short-run, maybe we should move it into the 2025 repo?